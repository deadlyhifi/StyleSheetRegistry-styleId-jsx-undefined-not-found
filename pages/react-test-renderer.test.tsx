import * as React from "react";
import ReactTestRenderer from "react-test-renderer";
import Home from ".";

it("renders as expected", () => {
  const container = ReactTestRenderer.create(<Home />);

  expect(container).toMatchSnapshot();
});
