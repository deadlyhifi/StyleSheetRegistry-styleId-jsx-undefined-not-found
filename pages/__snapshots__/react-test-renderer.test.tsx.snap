// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<ul
  className="jsx-undefined"
>
  <a
    className="jsx-undefined"
    data-qa="example1"
    href="https://www.example1.com"
    rel="noopener external"
    target="_blank"
    title="Example 1"
  >
    <li
      className="jsx-undefined"
    >
      <img
        alt="example1 logo"
        className="jsx-undefined"
        src="/static/icon.png"
      />
    </li>
  </a>
  <a
    className="jsx-undefined"
    data-qa="example2"
    href="https://www.example2.com"
    rel="noopener external"
    target="_blank"
    title="Example 2"
  >
    <li
      className="jsx-undefined"
    >
      <img
        alt="example2 logo"
        className="jsx-undefined"
        src="/static/icon.png"
      />
    </li>
  </a>
</ul>
`;
